slang-gfx.h

```diff
@@ -371,7 +371,7 @@ enum class PrimitiveType
 
 enum class PrimitiveTopology
 {
-    TriangleList,
+    TriangleList, TriangleStrip, PointList, LineList, LineStrip
```

Comment:
I assume we are consciously ignoring all uncommon GS- and tessellation-specific topologies for now?